Cockpit 263 (#492)
cockpit-project/cockpit-project.github.io: _posts/2022-02-16-cockpit-263.md

---
title: Cockpit 263
author: garrett
date: '2022-02-16'
tags: cockpit
slug: cockpit-263
category: release
summary: ''
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 263:

> ## The back and forward buttons in your browser now work as expected

Author: ```suggestion
## Back and forward browser buttons work as expected
```

Shorten headline

> Cockpit used to interfere with the browser history, and hitting the back button would frequently get into a loop, for example. This has been fixed and the back and forward button are now enabled again in Cockpit Client.

Author: ```suggestion
Cockpit used to interfere with browser history. Clicking the back button would frequently get into a loop. Browser history handling is now fixed, so navigating back works as expected.

 Additionally, back and forward buttons have been added to [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient), the desktop app to connect to Cockpit servers using SSH.
```

Author: I've reworded a bit, added a link to Cockpit Client, and explained what it is (as not everyone knows).

Author: We could add a screenshot of Cockpit Client's back/forward buttons here. But we should definitely at least update Cockpit Client's screenshots on Flathub when we can.

## Try it out

Cockpit 263 is available now:

* [For your Linux system](https://cockpit-project.org/running.html)

* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/263)
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)

> * [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)

Reviewer: Just to make sure, please update this to the actual bodhi URLs.

Reviewer 2: Done.

---
title: Cockpit 263
author: garrett
date: '2022-02-16'
tags: cockpit
slug: cockpit-263
category: release

> summary: ''

Author: ```suggestion
summary: 'Browser history fixes (back/forward buttons)'
```

We need a summary. How's this?
